frontend/src/components/AccountReceiveTable.vue

```diff
@@ -387,15 +400,37 @@ function useReceivedFundsTable(announcements: UserAnnouncement[], spendingKeyPai
   const isFeeLoading = ref(false);
   const isWithdrawInProgress = ref(false);
   const txHashIfEth = ref(''); // if withdrawing native token, show the transaction hash (if token, we have a relayer tx ID)
+  const vm = getCurrentInstance()!;
 
   // Define table columns
   const sortByTime = (a: Block, b: Block) => b.timestamp - a.timestamp;
   const toString = (val: BigNumber) => val.toString();
+
   const mainTableColumns = [
-    { align: 'left', field: 'timestamp', label: 'Date Received', name: 'date', sortable: true, sort: sortByTime },
-    { align: 'left', field: 'amount', label: 'Amount', name: 'amount', sortable: true, format: toString },
-    { align: 'left', field: 'from', label: 'Sender', name: 'from', sortable: true },
-    { align: 'left', field: 'receiver', label: 'Stealth Receiver', name: 'receiver', sortable: false },
+    {
+      align: 'left',
```

Comment:
_(arbitrary comment location to start a thread)_

> If key values are misspelled, the linter doesn't actually complain, so I still need to do an overview of the whole site to ensure no texts are missing.

Is this still a TODO before merging?

Reply:
Finished last bit of QA and translated few important error messages that were missed https://github.com/ScopeLift/umbra-protocol/pull/331/commits/55faea622331713489ac87bdcf61cb547326e082